### ISSUE TITLE
SockJS is seeking a new maintainer

### ISSUE THREAD

**Issue by brycekahle** [sockjs/sockjs-client#592]
Hi folks!

**TL;DR: Seeking new maintainer. Please reply if you are interested.**

Eight years ago I took over maintenance of SockJS from @majek. At the time, I was heavily involved in Javascript, Node.js, full stack and the like. I was actively using SockJS, primus, and other libraries in my day to day. It just made sense to help out.

I no longer work with Javascript in my day to day, and haven't for some time. I've done my best to keep up with issues and dependency updates for SockJS, but I think at this point I'm doing it a disservice by flying on autopilot mostly. I do think SockJS is a stable and mature library that doesn't need much in the way of functional changes. Ultimately, that is just my opinion and someone more actively involved in the Javascript/Node.js world may disagree.

Thus, I'm looking for someone to takeover maintaining SockJS. In terms of repos, it mostly consists of https://github.com/sockjs/sockjs-client, https://github.com/sockjs/sockjs-node, and https://github.com/sockjs/sockjs-protocol. There is not a large influx of issues, and a lot of them are duplicates of existing or closed issues.

If the new maintainer is interested in signing on to Tidelift, they are currently paying me ~$200USD/mo. You are not obligated to signup with them. I do pay for the `sockjs.org` domain, but that could be left to expire as it is just a redirect here.

If you want more details, or are interested in taking over, please reply. Thanks all!

**Comment by aclark4life** [comment]
@brycekahle I'll definitely do it if you can't find anyone that's more qualified 😄

**Comment by auvipy** [comment]
I am interested. I have been also maintaining a json:api javascript client for more then two years now.

**Comment by brycekahle** [comment]
@aclark4life @auvipy Can you both email me at sockjs@brycekahle.com ? It probably makes sense to have multiple folks with commit bits to reduce the burden.

**Comment by brycekahle** [comment]
@auvipy is taking over, so I'm closing this issue.
